### DISCUSSION TITLE
为什么升级到4.0.19就降不到4.0.11了

### DISCUSSION THREAD

**Discussion by pangxzi** [umijs/umi#9290] [Q&A]
![image](https://user-images.githubusercontent.com/52907799/189300051-630ba2ce-05c6-4c5b-9a8f-e29966eb134e.png)
![image](https://user-images.githubusercontent.com/52907799/189300062-7335abb6-3a11-47be-a24b-fb2f7ba1a77b.png)

我把node_modules删了 重新安装也是4.0.19

**Comment by HaishengLiang** [comment]
你需要把yarn.lock也删掉
或者不删除yarn.lock的情况下, 你需要先 remove 然后再重新 add
